docs/DeveloperGuide.md

```diff
@@ -443,6 +443,72 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
     
           Use case ends.
 
+**Use case: Check eligibility for assistance**
+
+**MSS:**
+
+1. User chooses which person to check eligibility for assistance
+2. SocialBook displays what schemes the person is eligible for
+
+    Use case ends.
+
+**Extensions:**
+
+* 1a. Given index is invalid.
+    * 1a1. SocialBook informs user that the index is invalid
+      Use case resumes at step 1.
```

Comment:
Can change `at` to `from` as thats the format used in the notes too. Should resume from step 2 after changes above made.